src/app/components/dialog-profile/dialog-profile.component.html

```diff
@@ -0,0 +1,87 @@
+<div class="dialog-registration">
+  <form
+    ngClass="form-registration"
+    [formGroup]="registrationGroup"
+    (ngSubmit)="onSubmit()"
+  >
+    <div class="form-item form-login">
+      <label for="login"> Login </label>
+      <input name="login" type="text" formControlName="login" />
+      <div class="alert alert-danger"></div>
+    </div>
+    <div class="form-item from-name">
+      <label for="name"> Name </label>
+      <input name="name" type="text" formControlName="name" required />
+      <div class="alert alert-danger">
+        <div *ngIf="name!.invalid && (name!.dirty || name!.touched)">
+          <div *ngIf="name!.errors?.['required']">Name is required.</div>
```

Comment:
Can be transformed to getting error message by error field name

`*ngFor="let errorKey of Object.keys(name.errors)">
<div>errorMessages[errorKey]</div>`